app/src/main/java/org/mifos/selfserviceapp/ui/fragments/HomeFragment.java

```diff
@@ -0,0 +1,311 @@
+package org.mifos.selfserviceapp.ui.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+import org.mifos.selfserviceapp.R;
+import org.mifos.selfserviceapp.models.accounts.loan.LoanAccount;
+import org.mifos.selfserviceapp.models.accounts.savings.SavingAccount;
+import org.mifos.selfserviceapp.models.accounts.savings.SavingsWithAssociations;
+import org.mifos.selfserviceapp.models.accounts.share.ShareAccount;
+import org.mifos.selfserviceapp.models.client.Client;
+import org.mifos.selfserviceapp.presenters.HomePresenter;
+import org.mifos.selfserviceapp.presenters.SavingAccountsDetailPresenter;
+import org.mifos.selfserviceapp.ui.activities.base.BaseActivity;
+import org.mifos.selfserviceapp.ui.fragments.base.BaseFragment;
+import org.mifos.selfserviceapp.ui.views.HomeView;
+import org.mifos.selfserviceapp.utils.Constants;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+import javax.inject.Inject;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import android.widget.Button;
+
+/**
+ * Created by michaelsosnick on 1/1/17.
+ */
+
+public class HomeFragment extends BaseFragment implements
+        SwipeRefreshLayout.OnRefreshListener, HomeView {
+
+    public static final String LOG_TAG = HomeFragment.class.getSimpleName();
+
+    @BindView(R.id.noAccountText)
+    TextView noAccountText;
+
+    @BindView(R.id.ll_error)
+    LinearLayout ll_error;
+
+    @BindView(R.id.noAccountIcon)
+    ImageView noAccountIcon;
+
+    @BindView(R.id.iv_profile_pic)
+    ImageView iv_profile_pic;
+
+    @Inject
+    HomePresenter homePresenter;
+
+    @Inject
+    SavingAccountsDetailPresenter savingPresenter;
+
+    View rootView;
+
+    @BindView(R.id.tv_client_name)
+    TextView tv_client_name;
+
+    @BindView(R.id.tv_savings)
+    TextView tv_savings;
+
+    @BindView(R.id.tv_loans)
+    TextView tv_loans;
+
+    @BindView(R.id.tv_shares)
+    TextView tv_shares;
+
+    @BindView(R.id.application_button)
+    Button application_button;
+
+    @BindView(R.id.saving_button)
+    Button saving_button;
+
+    @BindView(R.id.loan_button)
+    Button loan_button;
+
+    @BindView(R.id.share_button)
+    Button share_button;
+
+    @BindView(R.id.transaction_button)
+    Button transaction_button;
+
+    @BindView(R.id.transfer_button)
+    Button transfer_button;
+
+    @BindView(R.id.payment_button)
+    Button payment_button;
```

Comment:
Follow the naming convention, Prefix of the view like for Button it's `btn` and for TextView it's `tv` and them add the postfix of the name of the view that you wanna name.
Check out this [file](https://github.com/openMF/self-service-app/blob/development/app/src/main/java/org/mifos/selfserviceapp/ui/fragments/LoanAccountsDetailFragment.java) for example.